packit_service/worker/tasks.py

```diff
@@ -40,3 +49,43 @@
 @celery_app.task(name="task.steve_jobs.process_message")
 def process_message(event: dict, topic: str = None) -> Optional[dict]:
     return SteveJobs().process_message(event=event, topic=topic)
+
+
+@celery_app.task(
+    bind=True,
+    name="task.babysit_copr_build",
+    # 120 minutes
+    default_retry_delay=120 * 60,
+)
+def babysit_copr_build(self, build_id: int):
+    """ check status of a copr build and update it in DB """
+    builds = CoprBuild.get_all_by_build_id(build_id)
+    if builds:
+        copr_client = CoprClient.create_from_config_file()
+        build_copr = copr_client.build_proxy.get(build_id)
+
+        if not build_copr.ended_on:
+            # still in progress, retry
+            self.retry(countdown=60)  # try again in 60 seconds
+
+        # copr doesn't tell status of how a build in the chroot went:
+        #   https://bugzilla.redhat.com/show_bug.cgi?id=1813227
+        for build in builds:
+            event = CoprBuildEvent(
+                topic=str(FedmsgTopic.copr_build_finished),
+                build_id=build_id,
+                build={},
+                chroot=build.target,
+                status=1 if build_copr.state == COPR_SUCC_STATE else 2,
```

Comment:
Some constants for the `1` and `2`?

Reply:
this is coming from copr client, I can do constants